### DISCUSSION TITLE
CDXML Parser

### DISCUSSION THREAD

**Discussion by liushili0319** [rdkit/rdkit#6528] [Development]
I am trying to parse a chemical generation route using Rdkit, and I want to know if Rdkit can parse the complete route without losing information

**Comment by bp-kelley** [comment]
In the latest release, the CDXML Parser is fairly robust.  There are a few features we are currently missing, like brackets for repeats.

Note that the chemdraw format is quite complicated and has many added features we will most likely never support, so we will always loose information such as annotations/drawings/etc.
